src/tests/api/BavHappyPath.test.ts

```diff
@@ -56,9 +58,31 @@ describe("Test BAV End Points", () => {
         const authCode = authResponse.data.authorizationCode;
         const authRepeatResponse = await authorizationGet(origSessionId);
         const authRepeatResponseCode = authRepeatResponse.data.authorizationCode;
-        expect(authCode).not.toEqual(authRepeatResponseCode); 
+        expect(authCode).not.toEqual(authRepeatResponseCode);
+    });
+
+    it.skip("BAV CRI /userinfo Happy Path", async () => {
+        // const documentSelectionResponse = await postDocumentSelection(docSelectionData, sessionId);
+
+        // const authResponse = await authorizationGet(sessionId);
+
+        // const tokenResponse = await tokenPost(authResponse.data.authorizationCode.value, authResponse.data.redirect_uri );
+
+        // const userInfoResponse = await userInfoPost("Bearer " + tokenResponse.data.access_token);
+        const userInfoResponse = await userInfoPost("Bearer eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImYzODA1Y2UyLWQzNjYtNDY3ZC1iYzhkLWMzMTc4MGFmNTQyYyJ9.eyJzdWIiOiIzZjUyYzczYi05OGM2LTQxZjMtOTA5Zi1hMTAwOTFjODI3YzgiLCJhdWQiOiJodHRwczovL3Jldmlldy1iYXYuZGV2LmFjY291bnQuZ292LnVrIiwiaXNzIjoiaHR0cHM6Ly9yZXZpZXctYmF2LmRldi5hY2NvdW50Lmdvdi51ayIsImV4cCI6MTY5OTYzMjUxMH0.hlPhPY50RL2nNJt4HU9T3j8I99leRYwpy0EtlqIrJRy-GcWFhR0DxMsWw6uH3RxioVyu9iGJOw2WNuqaOmXJyg");
```

Comment:
This should not be hard coded, we'll get our Bearer token from the /token response